Jenkinsfile

```diff
@@ -128,7 +128,7 @@ def getBuildNum() {
     dir('.buildnumber') {
       try {
         sh 'git clone --depth 5 git@github.com:powerhome/nitro-buildnumber.git .'
-        buildNum = sh(returnStdout: true, script: './increment PlaybookSwift-version').trim().toInteger()
+        buildNum = sh(returnStdout: true, script: './increment playbook-swift-version').trim().toInteger()
```

Comment:
@isismsilva there will need to be a PR to [rename this file](https://github.com/powerhome/nitro-buildnumber/blob/master/PlaybookSwift-version) and delete the current one named `playbook-swift-version` else you will now be at version 2 for your builds instead of version `1162` (this PR). Just a heads up.

Reply:
Hey. Thank you, here is the PR for it. 
https://github.com/powerhome/nitro-buildnumber/pull/4/files